src/vm.rs

```diff
@@ -27,7 +27,7 @@ use crate::vm_cpu::{
 use crate::vm_pages::Error as VmPagesError;
 use crate::vm_pages::{
     ActiveVmPages, AnyVmPages, InstructionFetchError, PageFaultType, VmPages, VmPagesRef,
-    VmRegionList, TVM_REGION_LIST_PAGES, TVM_STATE_PAGES,
+    VmRegionList, TVM_REGION_LIST_PAGES,
```

Comment:
Unrelated nit: In the commit title, "pages numbers" should be "page numbers".

Reply:
Fixed. :)